phys2bids/physio_obj.py

```diff
@@ -228,6 +228,21 @@ class BlueprintInput():
     def __init__(self, timeseries, freq, ch_name, units, trigger_idx,
                  num_timepoints_found=None, thr=None, time_offset=0):
         """Initialise BlueprintInput (see class docstring)."""
+        trigger_names_list = ["trig", "trigger", "TRIGGER", "Trigger"]
+        if trigger_idx == 0:
+            name_not_found = True
+            LGR.warning('User did not input chtrig. Trying to find in auto mode by name')
```

Comment:
Could you add periods to the ends of the strings too? For consistency- since there are multiple sentences and all of the other sentences have punctuation.